uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Form/CreateUserRecordTaskForm.php

```diff
@@ -218,4 +224,19 @@ public function validateInput($fields) {
     return empty($errors) ? TRUE : $errors;
   }
 
+  /**
+   * Restores user account after delete
```

Comment:
Perhaps add a bit more flesh to this. A little explanation about what happens when a contact is soft deleted and why we need this method would be helpful.

Reply:
👌 Comments will be updated.